src/components/CharacterForm/CharacterForm.js

```diff
@@ -149,6 +196,8 @@ const CharacterForm = ({ setOpen }) => {
               label="Height"
               placeholder="Enter height"
               type="number"
+              rules={rules.height}
+              errors={errors.height}
```

Comment:
Pay attention to your naming, "errors" is no longer an appropriate name for this prop, it should be `error`